web_app_drawer/views/web.xml

```diff
@@ -130,109 +130,149 @@
 		</t>
 	</template>
 
-	<template id="web.menu_secondary">
+		</xpath>
 		
-		<t t-foreach="menu_data['children']" t-as="menu">
-			<ul style="display: none" class="oe_secondary_menu nav navbar-nav" t-att-data-menu-parent="menu['id']">
-				<t t-call="web.menu_secondary_submenu" />
-			</ul>
-		</t>
-	
-	</template>
-
-	<template id="web.menu_secondary_submenu">
-		<t t-foreach="menu['children']" t-as="menu">
-			<t t-if="menu['children']">
-				<li t-attf-class="{{ 'dropdown-submenu' if submenu else '' }}">
-					<t t-if="submenu">
-						<a href="#" class="dropdown-toggle">
-							<t t-esc="menu['name']" />
-						 </a>
-					</t>
-					<t t-if="not submenu">
-						<a href="#"
-						   class="dropdown-toggle"
-						   data-toggle="dropdown"
-						   role="button"
-						   aria-haspopup="true"
-						   aria-expanded="false"
-						   >
-							<t t-esc="menu['name']" />
-							<span class="caret" />
-						 </a>
-					</t>
-					<ul t-if="menu['children']"
-						t-attf-class="dropdown-menu oe_secondary_submenu"
-						>
-						<t t-call="web.menu_secondary_submenu">
-							<t t-set="submenu" t-value="True" />
-						</t>
-					</ul>
-				</li>
-			</t>
-			<t t-if="not menu['children']">
-				<li>
-					<t t-call="web.menu_link" />
-				</li>
-			</t>
-		</t>
-	</template>
+    </template>
 
-	<template id="web.menu_link">
+    <template id="menu_secondary"
+			  inherit_id="web.menu_secondary"
+			  name="App Drawer - Secondary Menu"
+			  >
```

Comment:
![tabsspacesboth](https://cloud.githubusercontent.com/assets/973709/17922534/dc3f4e5a-69df-11e6-857f-72b531601c31.png)

Please don't make us get the baseball bat out. :laughing: